components/infieldbutton/themes/spectrum-two.css

```diff
@@ -23,9 +23,9 @@
 		--spectrum-infield-button-stacked-top-border-radius-start-start: var(--spectrum-infield-button-border-radius-reset);
 		--spectrum-infield-button-stacked-bottom-border-radius-end-start: var(--spectrum-infield-button-border-radius-reset);
 
-		--spectrum-infield-button-background-color: var(--spectrum-gray-50);
-		--spectrum-infield-button-background-color-hover: var(--spectrum-gray-100);
+		--spectrum-infield-button-background-color: var(--spectrum-gray-100);
+		--spectrum-infield-button-background-color-hover: var(--spectrum-gray-200);
 		--spectrum-infield-button-background-color-down: var(--spectrum-gray-200);
-		--spectrum-infield-button-background-color-key-focus: var(--spectrum-gray-100);
+		--spectrum-infield-button-background-color-key-focus: var(--spectrum-gray-200);
```

Comment:
These changes, along with the stepper changes, are not reflected in Figma yet. They're working on updating them, but decided to have the stepper buttons' background color match the s2 picker background color.